acl-lib/src/cmr/acl/core.clj

```diff
@@ -10,6 +11,7 @@
    [cmr.common.date-time-parser :as dtp]
    [cmr.common.services.errors :as errors]
    [cmr.common.util :as util]
+   [cmr.transmit.access-control :as access-control]
```

Comment:
cmr.access-control.services.auth-util has calls to cmr.acl.core/get-permitting-acls which calls context->sids.